mpa/cls/stage.py

```diff
@@ -35,7 +35,8 @@ def configure(self, model_cfg, model_ckpt, data_cfg, training=True, **kwargs):
             else:
                 cfg.model = copy.deepcopy(model_cfg.model)
 
-        if cfg.model.pop('task', None) != 'classification':
+        cfg.model_task = cfg.model.pop('task', 'classification')
```

Comment:
Do you have any reason to use `cfg.model_task` instead of `cfg.task_type`?

Reply:
Well, no specific reason, but I think task_type is more like 'instance-segmentation', 'multilabel',,...